admin/routes/check-started.js

```diff
@@ -1,10 +1,15 @@
 const express = require('express')
 const router = express.Router()
 const { checkStarted } = require('../controllers/check-started')
+const winston = require('winston')
 
 router.route('/').all((req, res) => {
-  if (req.method !== 'POST') return res.sendStatus(405)
+  if (req.method !== 'POST') {
+    winston.error(`MTC-API: ${req.url}: req.method is not POST - res.sendStatus(405) - ${JSON.stringify(req.method)} `)
+    return res.sendStatus(405)
+  }
   checkStarted(req, res)
+  winston.info(`MTC-API: ${req.url}: request.body: ${JSON.stringify(req.body && req.body.length > 200 ? req.body.substring(0, 199) : '')}`)
```

Comment:
have you tested this? because i'm seeing the following output in your LogDNA....

`ERROR: req.body.substring is not a function ID:3c85bcbb-0343-4dee-b150-e9b0479349d2`
Also, i am not sure that you need to stringify it

Reply:
also, a dodgy request may not be JSON

Reply:
If I don't stringify I see [object][object]. 
I added (req.body && req.body.length > 0) to avoid that error you mentioned.

Reply:
I could write a small helper function to extend req.body validation, including if it's a JSON object or not.